salt/metalk8s/solutions/available.sls

```diff
@@ -0,0 +1,119 @@
+#
+# Management of Solution archives, mount/unmount and expose container images
+#
+# Relies on the `archives` key from /etc/metalk8s/solutions.yaml
+#
+
+{%- from "metalk8s/map.jinja" import repo with context %}
+
+include:
+  - metalk8s.repo.installed
```

Comment:
This one should be done at the end of the state but you already know :)

Reply:
I think it should not be called there as we change stuff impacting the external pillar, but the jinja is rendered before executing all this stuff... so it cannot work.

Reply:
`metalk8s.repo.installed` is no longer included here, I call it explicitly in the script, right after this sls.